src/components/modals/FioCreateHandleModal.tsx

```diff
@@ -15,15 +15,15 @@ import { ThemedModal } from '../themed/ThemedModal'
 
 interface Props {
   bridge: AirshipBridge<boolean>
-  createWalletsPromise: Promise<void>
+  createWalletsPromise?: Promise<void>
 }
 
 export const FioCreateHandleModal = (props: Props) => {
   const { bridge, createWalletsPromise } = props
 
   const theme = useTheme()
   const styles = getStyles(theme)
-  const isCreateWalletsInProgress = typeof createWalletsPromise === 'object' && createWalletsPromise instanceof Promise
+  const isCreateWalletsInProgress = typeof createWalletsPromise === 'object' && createWalletsPromise != null
```

Comment:
It seems like `createWalletsPromise != null` should be adequate, since the type system won't let us pass anything else.

I'm not sure how `isCreateWalletsInProgress` could have ever been false before, though, so maybe I'm wrong about this. Maybe just leave it if it's too tricky.